internal/daysteps/daysteps.go

```diff
@@ -10,6 +15,23 @@ var (
 
 func parsePackage(data string) (int, time.Duration, error) {
 	// ваш код ниже
+	sliseStrings := strings.Split(data, ",")
+	if len(sliseStrings) != 2 {
+		return 0, 0, fmt.Errorf("Ошибка")
+	}
+
+	var steps int
+	var err error
+	steps, err = strconv.Atoi(sliseStrings[0])
+	if steps <= 0 || err != nil {
+		return 0, 0, fmt.Errorf("Ошибка")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
3. Ошибку лучше начинать с маленькой буквы
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```

Reply:
НУжно везде поправить